yamls/jp/assets.yaml

```
@@ -4255,356 +4255,790 @@
     #   - [0x1106000, data, assets/objects/object_01106000/object_01106000]
   - [0x1106A60]
 
-  - name: object_01107000
-    type: bin # code
+  - name: tol_umb_01
+    dir: objects/tol_umb_01
+    type: code
     start: 0x1107000
     vram: 0x06000000
     exclusive_ram_id: segment_06
     compress: True
     align: 0x1000
-    # subsegments:
-    #   - [0x1107000, data, assets/objects/object_01107000/object_01107000]
+    subsegments:
+    symbol_name_format: $VRAM_$ROM
+    subsegments:
```

Please remove the duplicated `subsegments` entry
```suggestion
    symbol_name_format: $VRAM_$ROM
    subsegments:
```